ci-operator/jobs/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master-presubmits.yaml

```diff
@@ -62,6 +62,68 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )cinder-operator-build-deploy,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
always_run: false ?
IMO we should have one job with always_run: true only, to save resources. Since we already have build-deploy always running, we should add tempest to run only with comments, for now.
wdyt?

Reply:
I strongly recommend to keep this job running instead of the build and deploy one, since we can also test tempest. I know we don't have too much logs to debug yet, but keep waiting means we can add new bugs every change.

Reply:
wrt the add new bugs bit, just wanted to point out that this is more to do with the optional: true in ci-operator/config/openstack-k8s-operators/cinder-operator/openstack-k8s-operators-cinder-operator-master.yaml 
i guess you would need both for this then, always run and make it vote too
and if the others agree, switch the build-deploy job to optional instead and perhaps not always run but i think that is a wider discussion we should have